src/setup/rgw.py

```diff
@@ -584,6 +587,8 @@ def _logrotate_generic(conf: MappedConf):
             content = content.replace('TMP_LOG_PATH', log_file_path)
             with open(LOGROTATE_CONF, 'w') as f:
                 f.write(content)
+                f.close()
```

Comment:
Why did we closed the file descriptor explicitly?

Reply:
Python closes a file automatically, but It is a good practice to close an opened file.

Reply:
with open statement works as a context manager in Python.
No need to explicitly close the file.

Reply:
Removed.